hs/DrawResponse.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module DrawResponse where
+import           Text.Blaze ((!))
+import qualified Text.Blaze.Html5 as H
+import qualified Text.Blaze.Html5.Attributes as A
+import Happstack.Server -- need?
+import MakeElements
+import MasterTemplate
+import Scripts
+
+drawResponse :: ServerPart Response
+drawResponse =
+   ok $ toResponse $
+    masterTemplate "Courseography - Draw!"
+                [H.meta ! A.name "keywords"
+                        ! A.content "",
+                 timetableLinks
+                ]
+                (do
+                    header "draw"
+                    drawHtml
+
+                    button
+                    --modePanel
+                    theCanvas
+                )
+                timetableScripts
+
+
+drawHtml :: H.Html
+drawHtml = H.div ! A.id "aboutDiv" $ do
+  H.h1 "Draw a Graph"
+  H.p $ do
+    "Here you can create a graph yourself!"
```

Comment:
Unless you plan to do more in the `do` block, I don't think that you need to use one here.